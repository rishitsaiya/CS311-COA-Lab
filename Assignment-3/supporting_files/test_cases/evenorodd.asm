	.data
n:
	11
	.text
main:
	load %x0, $n, %x3
	divi %x3, 2, %x3
	beq %x0, %x31, even
	sub %x10, %x10, %x10
	addi %x10, 1, %x10
	end
even:
	sub %x10, $x10, %x10
	subi %x10, 1, %x10
	end
