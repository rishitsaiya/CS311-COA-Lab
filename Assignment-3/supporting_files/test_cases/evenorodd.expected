Hash of the Processor State = -224294686
